apps/omg/test/omg/state/core_test.exs

```diff
@@ -469,18 +469,6 @@ defmodule OMG.State.CoreTest do
     |> success?
   end
 
-  @tag fixtures: [:alice, :bob, :state_alice_deposit]
-  test "can spend after block is formed", %{alice: alice, bob: bob, state_alice_deposit: state} do
```

Comment:
Not really possible to run this test. The test relies on utxo staying in memory.
The test description itself indicates it to be integration tests. As a result leaving this to be covered by cabbage.